docs/static/tutorial-tool/tutorial.ts

```diff
@@ -0,0 +1,462 @@
+/// <reference path="../../../node_modules/monaco-editor/monaco.d.ts" />
```

Comment:
maybe add a comment to the pxt tutorial.ts noting that this file should be kept in sync?